bimsb/packages/staging.scm

```diff
@@ -2665,3 +2665,32 @@ to estimate cell type proportions in bulk RNA-seq data.")
 algorithm} (DSA) for extracting cell-type specific gene expression profiles
 from mixed tissue samples.")
       (license license:gpl2))))
+
+(define-public r-sanssouci
+  (let ((commit "5fe20a9aaf4ac637fa83d9cc73ff1c22de97ca6f")
```

Comment:
I like to add a comment here that explains quickly why we use this commit.

Reply:
Done. :-)